api/Element.json

```diff
@@ -5431,22 +5431,22 @@
               "version_added": null
             },
             "opera": {
-              "version_added": null
+              "version_added": "≤15"
```

Comment:
I'm positive this will go back far in Presto too, but a range for now means we'll come back to it.